test/passenger_test.rb

```diff
@@ -69,6 +73,130 @@
   end
 
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+    # let block for creating passenger - for last 2 tests
+    before do
+      @passenger = RideShare::Passenger.new(
+          id: 9,
+          name: "Merl Glover III",
+          phone_number: "1-602-620-2330 x3723",
+          trips: []
+      )
+
+      trip = RideShare::Trip.new(
+          id: 8,
+          passenger: @passenger,
+          start_time: Time.new(2016, 8, 8),
+          end_time: Time.new(2016, 8, 9),
+          rating: 5,
+          cost: 32.5,
+          driver: RideShare::Driver.new(
+              id: 1,
+              name: "Jill",
+              vin: "12345678901234567",
+              )
+      )
+
+      @passenger.add_trip(trip)
+      @passenger.add_trip(trip)
+    end
+
+    it "returns 0 for empty array" do
+      passenger = RideShare::Passenger.new(
+          id: 9,
+          name: "Merl Glover III",
+          phone_number: "1-602-620-2330 x3723",
+          trips: []
+      )
+      # create instance of passenger with empty array!
+      expect(passenger.net_expenditures).must_be_nil
+    end
+
+    it "return numeric value" do
+      expect(@passenger.net_expenditures).must_be_kind_of Numeric
+    end
+
+    it "returns sum of all trip costs for passenger" do
+      expect(@passenger.net_expenditures).must_equal 65
+    end
```

Comment:
Since `net_expenditures` returns a `Float`, you should use `must_be_close_to` just because floats have little rounding errors that crop up.  Avoid using `must_equal` with a float.